lib/ansible/modules/database/mongodb/mongodb_parameter.py

```diff
@@ -19,63 +19,70 @@
 DOCUMENTATION = '''
 ---
 module: mongodb_parameter
-short_description: Change an administrative parameter on a MongoDB server.
+short_description: Change an administrative parameter on a MongoDB server
 description:
     - Change an administrative parameter on a MongoDB server.
 version_added: "2.1"
 options:
     login_user:
         description:
-            - The username used to authenticate with
+            - The MongoDB username used to authenticate with.
+        type: str
     login_password:
         description:
-            - The password used to authenticate with
+            - The login user's password used to authenticate with.
+        type: str
     login_host:
         description:
-            - The host running the database
+            - The host running the database.
+        type: str
         default: localhost
     login_port:
         description:
-            - The port to connect to
+            - The MongoDB port to connect to.
         default: 27017
+        type: int
     login_database:
         description:
-            - The database where login credentials are stored
+            - The database where login credentials are stored.
+        type: str
     replica_set:
         description:
-            - Replica set to connect to (automatically connects to primary for writes)
-    database:
-        description:
-            - The name of the database to add/remove the user from
-        required: true
+            - Replica set to connect to (automatically connects to primary for writes).
+        type: str
     ssl:
         description:
-            - Whether to use an SSL connection when connecting to the database
+            - Whether to use an SSL connection when connecting to the database.
         type: bool
         default: 'no'
```

Comment:
```suggestion
        default: no
```
otherwise looks a bit like a string:)